isofit/core/fileio.py

```diff
@@ -349,9 +349,8 @@ def clear(self):
 class IO:
     """..."""
 
-    # Default ESD path
-    earth_sun_distance_path = os.path.join(
-        isofit.root, "data", "earth_sun_distance.txt"
+    earth_sun_distance_path = property(
+        lambda self: os.path.join(env.data, "earth_sun_distance.txt")
```

Comment:
@jammont A few checks are still failing since you initialize the path to the Earth-Sun-Distance file as a `property` object, which is not callable.

Reply:
Should be resolved, but either way this line has been removed altogether so comment is deprecated